### PR TITLE
chart: support customized number of replicas of webhook and recovery-backend

### PR DESCRIPTION
**Pull request by derekbit**
Longhorn#5087

Signed-off-by: Derek Su <derek.su@suse.com>

### CODE REVIEW

File: chart/values.yaml [hunk 1]
```diff
@@ -218,6 +218,54 @@ longhornUI:
   #  label-key1: "label-value1"
   #  label-key2: "label-value2"
 
+longhornConversionWebhook:
+  replicas: 2
+  priorityClass: ~
+  tolerations: []
```
Comment (innobead) [inline chart/values.yaml:224]: Need to update toleration and node selector doc, like https://longhorn.io/docs/1.4.0/advanced-resources/deploy/node-selector/.

Comment (derekbit) [reply chart/values.yaml:224]: OK. https://github.com/longhorn/longhorn/issues/5089

File: chart/values.yaml [hunk 2]
```diff
@@ -218,6 +218,54 @@ longhornUI:
...
+  ## If you want to set tolerations for Longhorn UI Deployment, delete the `[]` in the line above
```
Comment (innobead) [inline chart/values.yaml:225]: The comment is for longhorn UI, so we probably need to correct it. (even I think this kind of info should be chart README instead)

File: chart/values.yaml [hunk 3]
```diff
@@ -218,6 +218,54 @@ longhornUI:
...
+  ## and uncomment this example block
+  # - key: "key"
+  #   operator: "Equal"
+  #   value: "value"
+  #   effect: "NoSchedule"
+  nodeSelector: {}
+  ## If you want to set node selector for Longhorn UI Deployment, delete the `{}` in the line above
```
Comment (innobead) [inline chart/values.yaml:232]: The same, the comment needs to fix, the same for recovery backend and admission webhook.

**Review by innobead — CHANGES_REQUESTED** [review]

File: chart/values.yaml (hunk 2 above)
Comment (derekbit) [reply chart/values.yaml]: Updated. Thank you.

**Review by innobead — APPROVED** [review]

**Comment by innobead** [comment]
@derekbit Please backport to 1.4.x.

**Pull request merged**